modules/nf-core/freyja/demix/main.nf

```diff
@@ -0,0 +1,41 @@
+process FREYJA_DEMIX {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::freyja=1.3.12"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/freyja:1.3.12--pyhdfd78af_0':
+        'quay.io/biocontainers/freyja:1.3.12--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(variants)
+    tuple val(meta2), path(depths)
+    path barcodes
+    path lineages
+
+    output:
+    tuple val(meta), path("*.demix.tsv"), emit: demix
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    freyja \\
+        demix \\
+        $args \\
+        --output ${prefix}.demix.tsv \\
```

Comment:
It's better to not enforce filenames on the user (only file extensions should be hardcoded). If a file name collision is possible you can add a check (see [here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6)

Reply:
You could loop over all those files and check them individually. I've done it before (I'll see if I can find it)